src/v2/client.tsx

```diff
@@ -62,6 +63,11 @@ async function setupClient() {
   } catch (error) {
     console.error("[v2/client.tsx] Error loading client:", error)
   }
+  try {
+    setupMarketing()
```

Comment:
We can just move this into the `setupClient` function up above

Reply:
Oh also, I think we should be `await`ing here on this function call as well

Reply:
That was somewhat intentional. We could await, but there's really nothing else happening. That said, it doesn't hurt one way or the other so I'll add it.

Reply:
Oh if its intentional and not needed lets just keep it parallel! np

Reply:
Given how this async closure executes it doesn't really matter. It'll be the last thing that executes here whether its await or not. The await is somewhat of a no-op. I added it because it's probably more correct behavior when scoped in the try/catch (otherwise I'd need a `.catch` on the promise call just incase it rejected).